utils/_features.py

```diff
@@ -1229,6 +1229,26 @@ def iast_source_body(test_object):
         pytest.mark.features(feature_id=175)(test_object)
         return test_object
 
+    @staticmethod
+    def iast_source_kafka_key(test_object):
+        """
+        IAST Source: Kafka message key
+
+        https://feature-parity.us1.prod.dog/#/?feature=179
+        """
+        pytest.mark.features(feature_id=179)(test_object)
```

Comment:
why can't I find this feature ID in FPD?

Reply:
Same for me, we should add them manually to the parity dashboard right?

Reply:
hehe, invent total

thank you @manuel-alvarez-alvarez for creating them and show me how to do it!